### PR TITLE
IGAPP-268: Update Documentation

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: docs/cicd.md [hunk 1]
```diff
@@ -23,19 +23,19 @@ If no branch is specified, main is used as default.
...
 
 See the table below for a more detailed overview:
 
-|Workflow|Checks|E2E tests|Delivery|integreat build|malte build|Version bump|
-|---|---|---|---|---|---|---|
-|commit|:heavy_check_mark:|:heavy_check_mark:|:x:|:x:|:x:|:x:|
-|bi_weekly_development_delivery|:heavy_check_mark:|:heavy_check_mark:|Development|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
-|bi_weekly_production_delivery|:heavy_check_mark:|:heavy_check_mark:|Production|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
-|api_triggered_delivery|:heavy_check_mark:|:heavy_check_mark:|Depends|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|
+|Wrokflow|Checks|E2E tests|integreat build|malte build|Version bump|
```
Comment (maxammann) [inline docs/cicd.md:33]:
```suggestion
|Workflow|Checks|E2E tests|integreat build|malte build|Version bump|
```

**Review by maxammann — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+[![Dependency Status](https://gemnasium.com/badges/github.com/Integreat/integreat-app.svg)](https://gemnasium.com/github.com/Integreat/integreat-app)
+versioning: ![versioning](https://img.shields.io/badge/calver-YYYY.MM.PATCH-22bfda.svg)
+
+# integreat-app
+## Contents
+* [Contributing](#contributing)
+* [Getting Started](#getting-started)
+* [Bug Reporting](#bug-reporting)
+* [Pull Requests](#pull-requests)
```
Comment (klinzo) [inline README.md:9]: Not sure if this has to be mentioned here separately as it is already part of contributing.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+**Testing with the live cms instance should be avoided. Instead, [please use the test cms](docs/technology-stack.md#test-cms).**
+
+If you want to know more about Integreat or if you want to join us, contact [Max](mailto:ammann@integreat-app.de),
+[Steffen](mailto:kleinle@integreat-app.de) or [our info mail](mailto:info@integreat-app.de) and visit our [website](https://integreat-app.de).
+
+## Getting Started
+### Prerequisites
+* Rate our Integreat app in the [PlayStore](https://play.google.com/store/apps/details?id=tuerantuer.app.integreat)
+and the [Apple App Store](https://apps.apple.com/ae/app/integreat/id1072353915).
+* Install [nodejs](https://nodejs.org/). At least v6 is required, but we recommend the v12 LTS.
```
Comment (klinzo) [inline README.md:27]:
```suggestion
* Install [nodejs](https://nodejs.org/). At least v10 is required, but we recommend the v12 LTS.
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+* native:
+    * Take a look at native/package.json to show all available npm scripts.
+    * Run `yarn start` to start the bundler.
+    
+    *Note: For editing native Java code for Android IntelliJ with the Android extension is recommended. For native Swift/Objective-C code [XCode](https://developer.apple.com/xcode/) is required.*
+* locales
+    
+
+#### Additional Configuration
+* Mark the *src* and *www* (web-only) folder as *Source directory*.
```
Comment (klinzo) [inline README.md:68]:
```suggestion
* Mark the *src* and *www* (web-only) folders as *Source directory*.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    * Run `yarn start` to start a local webpack dev server.
+    * Start coding :)
+* native:
+    * Take a look at native/package.json to show all available npm scripts.
+    * Run `yarn start` to start the bundler.
+    
+    *Note: For editing native Java code for Android IntelliJ with the Android extension is recommended. For native Swift/Objective-C code [XCode](https://developer.apple.com/xcode/) is required.*
+* locales
+    
+
```
Comment (klinzo) [inline README.md:66]:
```suggestion
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+* Settings > Languages & Frameworks > JavaScript and
+    * Choose *Flow* as Language version
+    * Set *Flow package or executable* to *<project_dir>/node_modules/flow-bin*
+* [optional] Associate the *\*.snap* files with the file type *JavaScript*.
+* [optional] Install the following plugins:
+    * [Styled Components](https://plugins.jetbrains.com/plugin/9997-styled-components--styled-jsx/)
+    * [EJS](https://plugins.jetbrains.com/plugin/index?xmlId=com.jetbrains.lang.ejs) (for web)
+    * [Ruby](https://plugins.jetbrains.com/plugin/1293-ruby) (if working with Fastlane)
+* [optional] Configure Linux environment on Windows: [WSL Setup](docs/wsl-setup.md)
```
Comment (klinzo) [inline README.md:79]:
```suggestion
* [optional] Configure Linux environment on Windows: [WSL Setup](docs/wsl-setup.md)

```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+## Bug reporting
+You can [view our issues](https://issues.integreat-app.de/projects/IGAPP) or
+ [create new ones](https://issues.integreat-app.de/secure/CreateIssue!default.jspa) on our jira.
+
+## Pull requests
+Please take a look at our [conventions](docs/conventions.md).
+
+To merge a pull request,
+* at least two approvals are required.
+* tests, linting and flow have to succeed.
```
Comment (klinzo) [inline README.md:93]: Is this a duplicate of the pull request section in `.github`?

File: docs/contributing.md [hunk 8]
```diff
@@ -35,12 +35,12 @@ To merge a pull request, the following is necessary:
 Pull requests with changes that are visible to our end users should always include one (or more) release note describing the changes.
 For other changes, these release notes are optional, but make sure to set the flag `show_in_stores` to false.
 
-To add a release note:
-* Have a look at the [template](../release-notes/ReleaseNoteTemplate.yml).
-* Add a new release note to the [unreleased directory](../release-notes/unreleased).
+To add a release note (exemplary on native):
+* Have a look at the [template](native/release-notes/ReleaseNoteTemplate.yml).
+* Add a new release note to the [unreleased directory](native/release-notes/unreleased).
```
Comment (klinzo) [inline docs/contributing.md:40]: I think we only have one release notes dir in the root of the monorepo, don't we?

**Review by klinzo — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (maxammann) [reply README.md:93]: I think yeah, we can remove this section imo though.

**Pull request merged**